client.go

```diff
@@ -747,3 +787,7 @@ func (c *ovndb) sbGlobalAdd(options map[string]string) (*OvnCommand, error) {
 func (c *ovndb) sbGlobalDel() (*OvnCommand, error) {
 	return c.sbGlobalDelImp()
 }
+
+func (c *ovndb) chassisPrivateAdd(name string, external_ids map[string]string) (*OvnCommand, error) {
```

Comment:
Please move chassisPrivateAdd naear L433 to avoid scattering api calls in different places in function

Reply:
@noah8713 **chassisPrivateAdd** api is only used as helper function in unit-tests so that was the reason I added in the section of api's which are used as helpers for unit tests.

Reply:
Thanks. However, any api exposed in client should follow CRUD order. Also please check as CI is failing too:
https://travis-ci.org/github/eBay/go-ovn/builds/758743285?utm_source=github_status&utm_medium=notification

```
./chassis_private_test.go:26:23: ovndbapi.chassisPrivateAdd undefined (type Client has no field or method chassisPrivateAdd)
note: module requires Go 1.12FAIL	github.com/ebay/go-ovn [build failed]
```

Reply:
This still shows failure in CI. Did you push the fix?

Reply:
Please move this helper function with CRUD calls in line

Reply:
Not yet, waiting for the resolution of the discussion on LatestSBTablesOrder